src/views/labs/LabView/components/LabLeaderboard.vue

```diff
@@ -0,0 +1,162 @@
+<template>
+    <tr class="player-card">
+      <td>
+        <p class="rank">
+          #{{ rank + 1 }}
+        </p>
+      </td>
+      <td style="shrink">
+        <img v-if="imageLink" class="rounded-circle player-image" :src="imageLink" />
+        <img v-else class="rounded-circle player-image" src="@/assets/front-page/img/icon_user.png" />
+      </td>
+      <td>
+        <div class="player-name">
+          <a v-bind:href="'/players/' + player.uid">
+            {{ player.name }}
+          </a>
+        </div>
+       </td>
+      <td>
+        <top>
+        <a v-bind:href="puzzleLink">
+          {{ player.puztitle }}
+        </a>
+        </top>
+      </td>
+      <td>
+        <a v-bind:href="solutionLink"> 
+          {{ player.title }} 
+        </a>
+      </td>
+      <td>
+        <div class="score">
+          {{player["synthesis-score"]}}
+        </div>
+      </td>
+    </tr>
+</template>
+
+<script lang="ts">
+
+  import { Component, Prop, Vue } from 'vue-property-decorator';
+  import {
+    PUZZLE_ROUTE_PREFIX,
+    PUZZLE_ROUTE_BROWSE_PREFIX,
+    PUZZLE_ROUTE_SOLUTIONS_PREFIX,
+  } from '@/utils/constants';
+
+@Component({
+    components: {},
+})
+  export default class LabLeaderboard extends Vue {
+
+    @Prop({required: true}) readonly player!: any;
```

Comment:
While I'm having a hard time thinking of one, is there perhaps a better name for this than `player`? `player.puztitle` feels a bit awkward, since the player doesn't "have a" puzzle title.`rankedSolution` maybe?